cpu/native/include/periph_cpu.h

```diff
@@ -96,6 +96,52 @@ typedef enum {
 } gpio_flank_t;
 
 /** @} */
+#else
```

Comment:
shouldn't there be some GPIO_MOCK module to activate this?

```suggestion
#elif MODULE_GPIO_MOCK
```

Reply:
done